admin/fleetcommander/utils.py

```diff
@@ -115,3 +116,15 @@ def merge_settings(a, b):
         result[domain] = [index[key] for key in index]
 
     return result
+
+
+def get_free_port():
```

Comment:
We don't have to. We can use the Soup way in this case. The code is still needed in libvirtcontroller for random port in tunnel connection